### ISSUE TITLE
[WebToolsE2E][Aspire] Installing 9.0 template packs will override the available 8.2.x template packs.

### ISSUE THREAD

**Issue by v-elenafeng** [dotnet/aspire#6247]
**REPRO STEPS**
1.  Install SDK 9.0 RC2
2.  Install Aspire 8.2.1
3.  dotnet nuget add source "dotnet8"
4.  Install Aspire 9.0 template, net9

**ACTUAL**
Installing 9.0 template packs will override the available 8.2.x template packs.
![Image](https://github.com/user-attachments/assets/c7d8490e-921b-4f28-ac3b-0e44b3c11731)

**EXPECTED**
We wouldn't expect the existing 8.0 template packs be override by installing Aspire 9.0 (net9) template packs? Installing Aspire 9 should not affect Aspire 8?

**Comment by balachir** [comment]
I believe this is By Design now, the message is accurate that the 9.0 template pack is overriding the 8.2.x template pack. I'll go ahead and close this. 

@joperezr

**Comment by joperezr** [comment]
That is correct, this is expected and by design. Thanks for reporting it though 😃
